grammar.pegjs

```diff
@@ -183,14 +183,17 @@ geometry_point
       distance:(
         value_separator quotation_mark operator:distance_operator quotation_mark
         name_separator value:number_positive
-        { return { [operator]: value }; }
+        { 
+          var result = {};
+          result[operator] = value;
+          return result;
+        }
       )*
     end_object
     {
-      return {
-        "$geometry": geometry,
-        ...(distance ? distance : {})
-      };
+      var result = distance;
```

Comment:
Whoops my bad: this might need to be
```
  var result = distance || {};
```
to be the same as the old code :)

Reply:
Oh good catch! Glad I asked for review :)